app/code/Magento/InventoryCatalog/Plugin/InventoryApi/StockSourceLinksDelete/PreventDeleteDefaultStockLinksPlugin.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Plugin\InventoryApi\StockSourceLinksDelete;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\InventoryApi\Api\StockSourceLinksDeleteInterface;
+use Magento\InventoryCatalog\Api\DefaultStockProviderInterface;
+
+/**
+ * Prevent deleting links related to default stock.
+ */
+class PreventDeleteDefaultStockLinksPlugin
+{
+    /**
+     * @var DefaultStockProviderInterface
+     */
+    private $defaultStockProvider;
+
+    /**
+     * PreventDeleteDefaultStockLinksPlugin constructor.
```

Comment:
Pls remove this redundant comment (check all affected files)

Reply:
Removed.